presto-main/src/main/java/com/facebook/presto/cost/AggregationStatsRule.java

```diff
@@ -76,27 +94,59 @@ public static PlanNodeStatsEstimate groupBy(PlanNodeStatsEstimate sourceStats, C
             }));
         }
 
-        double rowsCount = 1;
+        double rowsCount = getDefaultAggregateSelectivityCoefficient(session);
```

Comment:
does this handle ungrouped aggregations properly now?  looks like we'd get a row count < 1 in this case.

Reply:
thanks for catching, no this is a bug, will fix

Reply:
Thanks. make sure this and the other issues here are also covered by tests.

Reply:
yes will do. I am running internal test, once all passed, the new commit will be pushed for you to re-review. thank you for your help!